### PR TITLE
Multi-conductor Rename

### PR DESCRIPTION
**Pull request by ccoffrin**
An update that works with https://github.com/lanl-ansi/PowerModels.jl/pull/324

@pseudocubic, please review.  Especially if I did the OpenDSS part correctly.

In terms of temp variables I changed `h` to `c` and `g` to `d`.

Once you are happy with this, squash merge the PowerModels PR and this one.

### CODE REVIEW

File: src/form/acp.jl [hunk 1]
```diff
@@ -102,23 +102,23 @@ p[t_idx] == z*(g*v[t_bus]^2 + (-g*tr-b*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[t_bus]-t[
...
-    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.phase_ids(pm)]
-    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.phase_ids(pm)]
-    va_fr = [var(pm, n, j, :va, f_bus) for j in PMs.phase_ids(pm)]
-    va_to = [var(pm, n, j, :va, t_bus) for j in PMs.phase_ids(pm)]
-    z = var(pm, n, h, :branch_z, i)
+function constraint_ohms_tp_yt_to_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractACPForm
+    p_to  = var(pm, n, c,  :p, t_idx)
+    q_to  = var(pm, n, c,  :q, t_idx)
+    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.conductor_ids(pm)]
+    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.conductor_ids(pm)]
```
Comment (pseudocubic) [inline src/form/acp.jl]: may want to change these `i` to `d` in `vm_fr` and `vm_to` as well so they don't collide with the `i` in the function arguments.

File: src/form/acp.jl [hunk 2]
```diff
@@ -75,25 +75,25 @@ p[f_idx] == z*(g/tm*v[f_bus]^2 + (-g*tr+b*ti)/tm*(v[f_bus]*v[t_bus]*cos(t[f_bus]
...
-    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.phase_ids(pm)]
-    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.phase_ids(pm)]
-    va_fr = [var(pm, n, j, :va, f_bus) for j in PMs.phase_ids(pm)]
-    va_to = [var(pm, n, j, :va, t_bus) for j in PMs.phase_ids(pm)]
-    z = var(pm, n, h, :branch_z, i)
+function constraint_ohms_tp_yt_from_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_fr, b_fr, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractACPForm
+    p_fr  = var(pm, n, c,  :p, f_idx)
+    q_fr  = var(pm, n, c,  :q, f_idx)
+    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.conductor_ids(pm)]
+    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.conductor_ids(pm)]
```
Comment (pseudocubic) [inline src/form/acp.jl]: may want to change these `i` to `d` in `vm_fr` and `vm_to` as well so they don't collide with the `i` in the function arguments.

File: src/form/dcp.jl [hunk 3]
```diff
@@ -5,33 +5,33 @@ Creates Ohms constraints (yt post fix indicates that Y and T values are in recta
...
-    va_fr = [var(pm, n, i, :va, f_bus) for i in PMs.phase_ids(pm)]
-    va_to = [var(pm, n, i, :va, t_bus) for i in PMs.phase_ids(pm)]
-    z = [var(pm, n, j, :branch_z, i) for j in PMs.phase_ids(pm)]
-
-    @constraint(pm.model, p_fr <= sum(-b[h,i]*(va_fr[i] - va_to[i] + vad_max[i]*(1-z[i])) for i in PMs.phase_ids(pm)) )
-    @constraint(pm.model, p_fr >= sum(-b[h,i]*(va_fr[i] - va_to[i] + vad_min[i]*(1-z[i])) for i in PMs.phase_ids(pm)) )
+function constraint_ohms_tp_yt_from_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_fr, b_fr, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractDCPForm
+    p_fr  = var(pm, n, c,  :p, f_idx)
+    va_fr = [var(pm, n, i, :va, f_bus) for i in PMs.conductor_ids(pm)]
+    va_to = [var(pm, n, i, :va, t_bus) for i in PMs.conductor_ids(pm)]
```
Comment (pseudocubic) [inline src/form/dcp.jl]: may want to change these `i` to `d` in `vm_fr` and `vm_to` as well so they don't collide with the `i` in the function arguments.

File: src/form/wr.jl [hunk 4]
```diff
@@ -36,23 +36,23 @@ p[f_idx] ==        g/tm*w_fr[i] + (-g*tr+b*ti)/tm*(wr[i]) + (-b*tr-g*ti)/tm*(wi[
...
+    @constraint(pm.model, p_fr ==  ( g_fr[c]+g[c,c]) * w[(f_bus, c)] +
+                                sum( g[c,i] * wr[(f_bus, f_bus, c, i)] +
+                                     b[c,i] * wi[(f_bus, f_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) +
+                                sum(-g[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                    -b[c,i] * wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
+    @constraint(pm.model, q_fr == -( b_fr[c]+b[c,c]) * w[(f_bus, c)] -
+                                sum( b[c,i] * wr[(f_bus, f_bus, c, i)] -
+                                     g[c,i] * wi[(f_bus, f_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) -
+                                sum(-b[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                     g[c,i] * wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
```
Comment (pseudocubic) [inline src/form/wr.jl]: may want to change these `i` to `d` in the constraint sums as well so they don't collide with the `i` in the function arguments.

File: src/form/wr.jl [hunk 5]
```diff
@@ -63,21 +63,21 @@ p[t_idx] ==        g*w_to[i] + (-g*tr-b*ti)/tm*(wr[i]) + (-b*tr+g*ti)/tm*(-wi[i]
...
+    @constraint(pm.model, p_to ==  ( g_to[c]+g[c,c]) * w[(t_bus, c)] +
+                                sum( g[c,i] * wr[(t_bus, t_bus, c, i)] +
+                                     b[c,i] *-wi[(t_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) +
+                                sum(-g[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                    -b[c,i] *-wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
+    @constraint(pm.model, q_to == -( b_to[c]+b[c,c]) * w[(t_bus, c)] -
+                                sum( b[c,i] * wr[(t_bus, t_bus, c, i)] -
+                                     g[c,i] *-wi[(t_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) -
+                                sum(-b[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                     g[c,i] *-wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
```
Comment (pseudocubic) [inline src/form/wr.jl]: may want to change these `i` to `d` in the constraint sums as well so they don't collide with the `i` in the function arguments.

File: src/io/opendss.jl [hunk 6]
```diff
@@ -20,16 +20,16 @@ function create_starbus(tppm_data::Dict, transformer::Dict)::Dict
...
     starbus["base_kv"] = 1.0
-    starbus["vmin"] = PMs.MultiPhaseVector(parse_array(0.9, nodes, phases))
-    starbus["vmax"] = PMs.MultiPhaseVector(parse_array(1.1, nodes, phases))
+    starbus["vmin"] = PMs.MultiConductorVector(parse_array(0.9, nodes, phases))
+    starbus["vmax"] = PMs.MultiConductorVector(parse_array(1.1, nodes, phases))
     starbus["name"] = "$(transformer["name"]) starbus"
-    starbus["vm"] = PMs.MultiPhaseVector(parse_array(1.0, nodes, phases))
-    starbus["va"] = PMs.MultiPhaseVector(parse_array(0.0, nodes, phases))
+    starbus["vm"] = PMs.MultiConductorVector(parse_array(1.0, nodes, phases))
+    starbus["va"] = PMs.MultiConductorVector(parse_array(0.0, nodes, phases))
```
Comment (pseudocubic) [inline src/io/opendss.jl]: `phases` should be changed to `nconductors` here

**Comment by ccoffrin** [comment]
Great, I'll make these updates and then merge everything.

**Pull request merged**